eng/Version.Details.xml

```diff
@@ -1,139 +1,139 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.5.24253.16">
-      <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>1741345c6399ae203d8f259fb12fb873dac5129d</Sha>
+    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.5.24304.3">
+      <Uri>https://github.com/dotnet/sdk</Uri>
+      <Sha>1fb3b2862a44d109ed755f96344fdd65830c15ca</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.5.24256.1" CoherentParentDependency="VS.Tools.Net.Core.SDK.Resolver">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.5.24280.5" CoherentParentDependency="VS.Tools.Net.Core.SDK.Resolver">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>84b33395057737db3ea342a5151feb6b90c1b6f6</Sha>
+      <Sha>de6897b3b71c1628329ab586ea3e4ecc6a5a2ab3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.5.292">
+    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.5.308">
```

Comment:
34.99.0-preview.5.308 is the tip of:
* https://github.com/xamarin/xamarin-android/commits/release/9.0.1xx-preview5/